### ISSUE TITLE
Provide opinionated install details and requirements for MacOS and WSL

### ISSUE THREAD

**Issue by cidrblock** [ansible/ansible-navigator#482]
##### ISSUE TYPE
 - Doc issue

##### SUMMARY

We should provide install instructions for MacOS and windows (WSL) that are opinionated and provide a known working safe path to get up and running

**Comment by ssbarnea** [comment]
Clearly that is needed. While installing navigator on macos worked fine and it did successfully pulled the container, I observed that the container is not really functional, as it reports no collections installed, no config at all, and some errors that were not really actionable:

```
WARNING
                                 ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                                 Errors were encountered while gathering the configuration:
                                 Error: statfs /var/folders/3q/pc1jcyjj3qqbj6_x7n4kbtqw0000gn/T/ansible-navigator_q3m5wn6v: no such file or directory
                                 Error: statfs /var/folders/3q/pc1jcyjj3qqbj6_x7n4kbtqw0000gn/T/ansible-navigator_q3m5wn6v/artifacts: no such file or directory
                                 The configuration could not be gathered.
                                 ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
                                                                                                                                                          Ok
```

**Comment by cidrblock** [comment]
Fixed in #593
